gui/src/components/Router.tsx

```diff
@@ -109,10 +109,12 @@ const Router = () => {
                                             />
                                         </ErrorBoundary>
                                     </Box>
-                                    {state.ackList.length && (
+                                    {state.ackList.length > 0 ? (
```

Comment:
I guess state.ackList.length is more efficient but you're right about the ternary operator